src/scripts/main.js

```diff
@@ -1,3 +1,31 @@
 'use strict';
 
-// write code here
+const parentElement = document.querySelector('ul');
+const employees = Array.from(document.querySelectorAll('li'));
```

Comment:
Ensure that the 'li' elements are present in the DOM before this line is executed. If the script runs before the DOM is fully loaded, 'employees' might be an empty array.